### PR TITLE
Nuke all

### PR DESCRIPTION
**Pull request by matzew**
* Remove all content, add a list of new repos

/hold

This needs to be merged, after:
* infra has disable the hooks for head 
* the code from `cmd` is reolcated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,16 @@
 # Knative Eventing Contrib
 
-[![go.dev reference](https://img.shields.io/badge/go.dev-reference-007d9c?logo=go&logoColor=white)](https://pkg.go.dev/github.com/knative/eventing-contrib)
-[![Go Report Card](https://goreportcard.com/badge/knative/eventing-contrib)](https://goreportcard.com/report/knative/eventing-contrib)
-[![Releases](https://img.shields.io/github/release-pre/knative/eventing-contrib.svg)](https://github.com/knative/eventing-contrib/releases)
-[![LICENSE](https://img.shields.io/github/license/knative/eventing-contrib.svg)](https://github.com/knative/eventing-contrib/blob/master/LICENSE)
-[![Slack Status](https://img.shields.io/badge/slack-join_chat-white.svg?logo=slack&style=social)](https://knative.slack.com)
+This repository has been replaced by a few separate repos where the development of the individual components are being developed:
```
Comment (mattmoor) [inline README.md:3]: Format markdown:
```suggestion
This repository has been replaced by a few separate repos where the development
of the individual components are being developed:
```

File: README.md [hunk 2]
```diff
@@ -1,23 +1,16 @@
...
-- Expose an ingress
+* [Integrate with Apache Camel-K](https://github.com/knative-sandbox/eventing-camel)
+* [Integrate with Apache CouchDB](https://github.com/knative-sandbox/eventing-couchdb)
+* [Integrate with Apache Kafka](https://github.com/knative-sandbox/eventing-kafka)
+* [Integrate with AWS SQS](https://github.com/knative-sandbox/eventing-awssqs)
+* [Integrate with Ceph](https://github.com/knative-sandbox/eventing-ceph)
+* [Integrate with GitHub](https://github.com/knative-sandbox/eventing-github)
+* [Integrate with GitLab](https://github.com/knative-sandbox/eventing-gitlab)
+* [Integrate with NATS Streaming](https://github.com/knative-sandbox/eventing-natss)
+* [Integrate with Prometheus](https://github.com/knative-sandbox/eventing-prometheus)
```
Comment (mattmoor) [inline README.md:13]: Format markdown:
```suggestion
- [Integrate with Apache Camel-K](https://github.com/knative-sandbox/eventing-camel)
- [Integrate with Apache CouchDB](https://github.com/knative-sandbox/eventing-couchdb)
- [Integrate with Apache Kafka](https://github.com/knative-sandbox/eventing-kafka)
- [Integrate with AWS SQS](https://github.com/knative-sandbox/eventing-awssqs)
- [Integrate with Ceph](https://github.com/knative-sandbox/eventing-ceph)
- [Integrate with GitHub](https://github.com/knative-sandbox/eventing-github)
- [Integrate with GitLab](https://github.com/knative-sandbox/eventing-gitlab)
- [Integrate with NATS Streaming](https://github.com/knative-sandbox/eventing-natss)
- [Integrate with Prometheus](https://github.com/knative-sandbox/eventing-prometheus)
```

**Review by mattmoor — COMMENTED** [review]
Produced via:
  `prettier --write --prose-wrap=always $(find -name '*.md' | grep -v vendor | grep -v .github | grep -v docs/cmd/)`

**Review by slinkydeveloper — APPROVED** [review]
The end of an era, full of CI failures. Long life to multi-repo!

**Comment by devguyio** [comment]
I feel it maybe better to just archive the repo after disconnecting from the CI. It preserves the last state the repo was at which might me beneficial. I'm not sure what are we gaining by deleting everything.

**Comment by matzew** [comment]
> I feel it maybe better to just archive the repo after disconnecting from the CI. It preserves the last state the repo was at which might me beneficial. I'm not sure what are we gaining by deleting everything.

well, most stuff as been already removed. E.g. github, ceph, etc.  Not sure how that helps in just archiving the current state, which is just `kafka` and the `cmd` - see my comments above for what is IMO needed before nuking

**Comment by antoineco** [comment]
I agree that it's cleaner to keep just a README at `HEAD`, and let people refer to the Git history / GitHub releases for historical revisions. And _then_ the repo can be archived.

**Pull request closed without merging**